plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -231,30 +227,41 @@ export class PersonState {
         }
     }
 
-    private updatedPersonProperties(personProperties: Properties): Properties {
-        const updatedProperties = { ...personProperties }
+    private updatedPersonProperties(person: Partial<Person>): Partial<Person> {
+        const updatedPerson = {
+            properties: person.properties || {},
+            properties_last_updated_at: person.properties_last_updated_at || {},
```

Comment:
This will increase load during ingestion significantly since we'll be constantly updating persons (e.g. due to geoip plugin).

I'm not sold on the debugging win claims here as we can just query the event history to reconstruct this.